src/promptflow/promptflow/_sdk/_service/utils/utils.py

```diff
@@ -36,6 +55,35 @@ def get_random_port():
         return s.getsockname()[1]
 
 
+def _get_process_by_port(port):
+    for proc in psutil.process_iter(["pid", "connections", "create_time"]):
+        try:
+            connections = proc.info["connections"]
+        except psutil.AccessDenied:
+            pass
+        else:
+            for connection in connections:
+                if connection.laddr.port == port:
+                    return proc
+
+
+def kill_exist_service(port):
+    proc = _get_process_by_port(port)
+    if proc:
+        proc.kill()
+
+
+def get_started_service_info(port):
+    service_info = {}
+    proc = _get_process_by_port(port)
+    if proc:
+        create_time = proc.info["create_time"]
+        process_uptime = datetime.now() - datetime.fromtimestamp(create_time)
+        service_info["uptime"] = str(process_uptime)
```

Comment:
I was thinking this info is get back from the /live api, but it's ok to be this in this PR